### DISCUSSION TITLE
Node and npm versions being used

### DISCUSSION THREAD

**Discussion by TheMohit2003** [ruxailab/RUXAILAB#380] [Q&A] [answered]
I have been working on dockerfile ( [issue](https://github.com/ruxailab/RUXAILAB/issues/360)) for a while and my script exits whenever the command sequence hits `npm install` , can you guys tell me what are node and npm  versions we are using so that i can use node image accordingly.
![image](https://github.com/ruxailab/RUXAILAB/assets/99909551/6c571462-eae9-40da-97d6-70467d678e0d)

**Comment by jvJUCA** [comment]
Hello @TheMohit2003 , 

As the main developer of the project, I'm using the following versions:

- npm: 8.19.4
- node: 16.20.1

I hope this helps address the issue you're encountering with the Dockerfile.

If you have any further questions, feel free to ask.

Best regards,
@jvJUCA

**Reply by KarinePistili** [reply, marked as answer]
It also works with:

- node: 20.11.1
- npm: 10.2.4
